pennylane_lightning/lightning_gpu/lightning_gpu.py

```diff
@@ -40,6 +40,21 @@
         DevPool,
     )
 
+    try:
+        # pylint: disable=no-name-in-module
+        from pennylane_lightning.lightning_gpu_ops import (
+            StateVectorMPIC128,
+            StateVectorMPIC64,
+            MeasurementsMPIC128,
+            MeasurementsMPIC64,
+            MPIManager,
+            DevTag,
+        )
+
+        MPI_SUPPORT = True
+    except ImportError:
+        MPI_SUPPORT = False
```

Comment:
So we always try to import the MPI build first? I wonder if we can tag the binary itself and just return T/F and use that instead of try/fail approach.

Let's track this as something to revisit.

Reply:
Let me add it to backlog.